tests/fvt/roles/non-cluster-tests/shell_execute/test_fvt_shell_execute.py

```
@@ -266,3 +266,31 @@ def test_shell_execute_mixed_output(role_dir, base_env, setup_test_env):
+        ["python3", str(role_dir / "main.py")], env=env, capture_output=True, text=True
+    )
+
+    assert result.returncode == 0, f"Role execution failed: {result.stderr}"
+
+    # Verify command output file exists
+    output_file = Path(base_env["ROLE_DIR"]) / "0-command.txt"
+    assert output_file.exists()
+
+    # Verify that /tmp/loopy directory was created
+    loopy_dir = Path("/tmp/loopy")
+    assert loopy_dir.exists(), f"Expected directory /tmp/loopy was not created"
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)